curriculum/challenges/english/01-responsive-web-design/css-ferris-wheel/part-021.md

```diff
@@ -0,0 +1,189 @@
+---
+id: 6140f4b5c1555a2960de1e5f
+title: Part 21
+challengeType: 0
+dashedName: part-21
+---
+
+# --description--
+
+Create another `@keyframes` rule with the name `cabins`. Use the same properties as your `@keyframes wheel`, but set the `transform` property of the `100%` selector to `rotate(-360deg)`.
```

Comment:
Given how late this project comes in the curriculum, do we want to be that specific in the instructions? Or do we want to rely more on the tests?

Reply:
As I say, I leave it up to you. I am happy with either, because I know to rely on the test text as much as necessary - something I think will be of much resistance for previous version Campers.

Reply:
Added a note.